ruby/ext/google/protobuf_c/protobuf.h

```diff
@@ -84,10 +84,11 @@ void Arena_Pin(VALUE arena, VALUE obj);
 // being collected (though in Ruby <2.7 is it effectively strong, due to
 // implementation limitations).
 
-// Adds an entry to the cache. The "arena" parameter must give the arena that
+// Adds an entry to the cache. Returns the pre-existing entry if there was one.
```

Comment:
`Tries to add a new entry to the cache, returning the newly installed value or the pre-existing entry.`

feels a bit more accurate to me

Reply:
Done.